cpu/native/include/cpu_conf.h

```diff
@@ -55,6 +55,11 @@ extern "C" {
  */
 #define NATIVE_ETH_PROTO 0x1234
 
+#if GNRC_PKTBUF_SIZE < 2048
+#   undef  GNRC_PKTBUF_SIZE
+#   define GNRC_PKTBUF_SIZE     (2048)
```

Comment:
> a) we're dealing with native here, so additional memory doesn't hurt and
> b) we would still have to touch all the examples and c) not erroring is
> friendlier.

while I totally agree with c),

- wasting memory and adding another copy of every packet is a waste even
on native, which I would rather not do just to comply to subjective
non-technical opinions

- "touching all examples" means touching one more example where someone
shows how to lower the pktbuf size, where native needs a totaly valid
exception